opm/grid/CpGrid.hpp

```diff
@@ -619,7 +619,27 @@ namespace Dune
                     const double* transmissibilities = nullptr,
                     int overlapLayers=1)
         {
-            return scatterGrid(wells, transmissibilities, overlapLayers);
+            return scatterGrid(1, wells, transmissibilities, overlapLayers);
+        }
+
+	// loadbalance is not part of the grid interface therefore we skip it.
+
+        /// \brief Distributes this grid over the available nodes in a distributed machine
+        /// \param ecl Pointer to the eclipse state information. Default: null
+        ///            If this is not null then complete well information of
+        ///            of the last scheduler step of the eclipse state will be
+        ///            used to make sure that all the possible completion cells
+        ///            of each well are stored on one process. This done by
+        ///            adding an edge with a very high edge weight for all
+        ///            possible pairs of cells in the completion set of a well.
+        /// \param The number of layers of cells of the overlap region (default: 1).
+        /// \warning May only be called once.
+        std::pair<bool, std::unordered_set<std::string> >
+        loadBalance(int edgeWeightsMethod, const std::vector<cpgrid::OpmWellType> * wells,
```

Comment:
I have two suggestions here to make the code more readable:
1. Please introduce a (documented) enum for the edgeWeightsMethod: `enum EdgeWeightMethods{ defaultbla, fancymethod};` and use that for the first parameter type and when calling `scatterGrid`.
2.  Please adjust the function documentation (shame one it is already broken, ouch)